pkgs/development/tools/faq/default.nix

```diff
@@ -0,0 +1,42 @@
+{ buildGoModule, pkgs, lib, fetchFromGitHub }:
+
+buildGoModule rec
+{
+  pname = "faq";
+  version = "0.0.8-rc";
```

Comment:
The current version seems `0.0.7` at the moment. Also I'd remove the `rc` suffix - it seems to me upstream uses the "pre-release" tag because it's in a early alpha stage.